.changeset/selfish-hounds-beg.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/selectable-search-input': major
+---
+
+pass down `components` props so that users can pass in custom select features
```

Comment:
```suggestion
We're introducing a new property (XXX) so consumers can customize the different parts of the select element in this component.
You can learn more about how this customization works in the `react-select` library [docs](https://react-select.com/components) we use under the hood
```

Reply:
updated hereL https://github.com/commercetools/ui-kit/pull/2831/commits/aa3cd25ca24a4ba0a42c335678e0d72d346457a3

Thank you